mergify_engine/actions/assign.py

```diff
@@ -43,6 +44,9 @@ def run(self, ctxt: context.Context, rule: rules.EvaluatedRule) -> check_api.Res
         if self.config["add_users"]:
             return self._add_assignees(ctxt, self.config["add_users"])
 
+        if self.config["remove_users"]:
```

Comment:
This won't work if you have both `add_users` and `remove_users` in the same action.